.github/workflows/cd.yml

```diff
@@ -0,0 +1,31 @@
+name: CD
+
+on:
+  push:
```

Comment:
What if we force push a tag to overwrite, would an image be rebuilt and updated?

Reply:
I believe so, yes